css/scss/_lists.scss

```
@@ -1,32 +1,44 @@
 ul {
-	list-style: none;
-	list-style-type: none;
+  list-style: none;
+  list-style-type: none;
 
-	li {
-		a {
-			position: relative;
-			padding-left: 1.25rem;
-			margin-bottom: 0.785rem;
-			display: block;
+  li {
+    a {
+      position: relative;
+      padding-left: 1.25rem;
+      margin-bottom: 0.785rem;
+      display: block;
 
-			&:hover {
-				&::before {
-					transform: translate(0.235rem, -0.25rem);
-				}
-			}
+      &:hover {
+        &::before {
+          transform: translate(0.235rem, -0.25rem);
+        }
+      }
 
-			&::before {
-				position: absolute;
-				top: 0.25rem;
-				left: 0;
-				display: block;
-				width: 0.685rem;
-				height: 0.685rem;
-				content: url('data:image/svg+xml;utf8,<svg aria-hidden="true" focusable="false" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512"><path fill="#{sansHashColor($black)}" d="M285.476 272.971L91.132 467.314c-9.373 9.373-24.569 9.373-33.941 0l-22.667-22.667c-9.357-9.357-9.375-24.522-.04-33.901L188.505 256 34.484 101.255c-9.335-9.379-9.317-24.544.04-33.901l22.667-22.667c9.373-9.373 24.569-9.373 33.941 0L285.475 239.03c9.373 9.372 9.373 24.568.001 33.941z"></path></svg>');
-				transform: translate(0, -0.25rem);
-				will-change: transform;
-				transition: transform 0.35s ease-in-out;
-			}
-		}
-	}
+      &::before {
+        position: absolute;
+        top: 0.25rem;
+        left: 0;
+        display: block;
+        width: 0.685rem;
+        height: 0.685rem;
+        content: url('data:image/svg+xml;utf8,<svg aria-hidden="true" focusable="false" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512"><path fill="#{sansHashColor($black)}" d="M285.476 272.971L91.132 467.314c-9.373 9.373-24.569 9.373-33.941 0l-22.667-22.667c-9.357-9.357-9.375-24.522-.04-33.901L188.505 256 34.484 101.255c-9.335-9.379-9.317-24.544.04-33.901l22.667-22.667c9.373-9.373 24.569-9.373 33.941 0L285.475 239.03c9.373 9.372 9.373 24.568.001 33.941z"></path></svg>');
+        transform: translate(0, -0.25rem);
+        will-change: transform;
+        transition: transform 0.35s ease-in-out;
+      }
+    }
+  }
+}
+
+#white-list {
+  li {
+    a {
+      color: white;
```

Missing $ for variable:
```suggestion
      color: $white;
```